game/src/environment/portal/Portal.gd

```diff
@@ -6,11 +6,14 @@ signal on_hit(isRed, health)
 export var health := 100.0
 onready var sprite: AnimatedSprite = $AnimatedSprite
 export var isRed: bool = false
-
+onready var health_bar = self.find_parent("Arena").get_node("UI/Control/HealthBar")
 
 func _ready() -> void:
 	sprite.play("red" if isRed else "blue")
-
+	if (isRed):
+		health_bar.init_red_portal_health = health
```

Comment:
i don't think it makes sense to do `init_red_portal_health`, should just be setting `red_portal_health`

Reply:
cleared up.